workloadDriver/src/main/java/com/vmware/weathervane/workloadDriver/common/core/loadControl/loadPath/SyncedFindMaxLoadPath.java

```diff
@@ -218,11 +219,18 @@ private IntervalCompleteResult initialRampIntervalComplete() {
 				}
 				getIntervalStatsSummaries().add(rollup);
 			}
-		} 
+		} 		
+		if (((getMaxPassHint() > 0) && (curUsers == getMaxPassHint())) || 
+				((getMaxPassHint() == 0) && !prevIntervalPassed) || 
+				((curUsers + getInitialRampRateStep()) > maxUsers)) {
+			result.setPassed(false);
+		} else {
+			result.setPassed(true);			
```

Comment:
Should this be result.setPassed(prevIntervalPassed)?  Is it possible prevIntervalPassed can be false in this path (from line 217/218)?

Reply:
No.  There are only the three cases in which we should set passed to false and thus move to findFirstMax.  In particular we want to set passed to true even if prevIntervalPassed is false when maxPassHint is non-zero.  When running with the hint, the initialRamp always ramps up to the value of maxPassHint, even if one of the intervals along the way fails response-time.